config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   namespace :api, defaults: { format: :json } do
     namespace :v1 do
+      resources :news
```

Comment:
Siempre debes procurar crear solo las rutas necesarias, en este caso por ejemplo:
```suggestion
      resources :news, only: %i[show create update destroy]
```
De todos modos vas a tener que crear una PR por ticket.